src/components/MyTemplate/MyTemplateListItem.tsx

```diff
@@ -1,11 +1,35 @@
 /* eslint-disable require-jsdoc */
-import React from "react";
-import { Flex, Box, Button, Text, Grid, Stack, Badge } from "@chakra-ui/react";
+import React, { useState, useRef } from "react";
+import { Flex, Box, Button, Text, Grid, Stack, Badge, useDisclosure } from "@chakra-ui/react";
 import moment from "moment";
 import { MdDeleteOutline, MdRemoveRedEye } from "react-icons/md";
 import Image from "next/image";
+import MyTemplateAlert from "./MyTemplateAlert";
+import { useUpdateTemplateMutation } from "@/redux/api/templateApi";
+import { IUpdateTemplate } from "@/interfaces/template";
 
 function MyTemplateListItem({ ...item }) {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const cancelRef = useRef();
```

Comment:
it can be declared in` MyTemplateAlert.tsx`

Reply:
I tried but it failed.